Description: Patches makefile and grokevy.py
 Use $(PREFIX) variable on Makefile
 Fixing config path
 Get correct python3 PATH

Author: Christophe Monniez <christophe.monniez@fccu.be>
Author: Samuel Henrique <samueloph@debian.org>
Author: Hugo Lefeuvre <hle@debian.org>
Index: grokevt/Makefile
===================================================================
--- grokevt.orig/Makefile
+++ grokevt/Makefile
@@ -4,9 +4,9 @@ PYTHON3_PATH:=`which python3`
 PYTHON_PATH:=`which python`
 
 # Installation prefixes.  Change to install elsewhere.
-PREFIX=/usr/local
+PREFIX=/usr
 BIN_PREFIX=$(PREFIX)/bin
-ETC_PREFIX=$(PREFIX)/etc/grokevt
+ETC_PREFIX=/etc/grokevt
 DOC_PREFIX=$(PREFIX)/share/doc/grokevt
 MAN_PREFIX=$(PREFIX)/man
 
@@ -35,8 +35,8 @@ install: all
 	cp -r $(BUILD_BIN)/* $(BIN_PREFIX)
 	cp -r $(BUILD_ETC)/* $(ETC_PREFIX)
 	cp -r $(BUILD_DOC)/* $(DOC_PREFIX)
-	if [ "x$(PYTHON_PATH)" != "x" ]; then $(PYTHON_PATH) grokevt-distutils.py install; fi
-	if [ "x$(PYTHON3_PATH)" != "x" ]; then $(PYTHON_PATH) grokevt-distutils.py install; fi
+	if [ "x$(PYTHON_PATH)" != "x" ]; then $(PYTHON_PATH) grokevt-distutils.py install --prefix $(PREFIX); fi
+	if [ "x$(PYTHON3_PATH)" != "x" ]; then $(PYTHON3_PATH) grokevt-distutils.py install --prefix $(PREFIX); fi
 	$(MAKE) -C doc install
 
 
Index: grokevt/lib/grokevt.py
===================================================================
--- grokevt.orig/lib/grokevt.py
+++ grokevt/lib/grokevt.py
@@ -688,7 +688,7 @@ class grokevtConfig:
 
 # This is only here to aide in debugging.
 # It will be overridden during a 'make install' below.
-PATH_CONFIG='/usr/local/etc/grokevt'
+PATH_CONFIG='/etc/grokevt'
 
 
 ################################################################################